network-interface/src/request/mod.rs

```diff
@@ -143,7 +143,7 @@ pub trait RequestCommon:
 pub trait RequestSerialize: RequestCommon {
     /// Serializes a request.
     /// A serialized request is composed of:
-    /// - A varint for the Type ID of the request
+    /// - A variant for the Type ID of the request
```

Comment:
```suggestion
    /// - A variable sized integer for the Type ID of the request
```

`varint` as in "variable sized integer".

Reply:
Thanks this was autocorrect mix-fixes